### DISCUSSION TITLE
data_start_values for maps

### DISCUSSION THREAD

**Discussion by SandraMarq** [open-sdg/open-sdg#2004] [Q&A]
Hello

The data_satar_value for maps doesn't work for me.

Add it to the corresponding file
![image](https://github.com/open-sdg/open-sdg/assets/118024145/e5a7fc22-80c1-4211-8111-c73a65d7117e)

Even so, it starts in alphabetical order
![image](https://github.com/open-sdg/open-sdg/assets/118024145/84cfc803-6475-49dc-8afe-f6a59bdd111c)

In addition to the fact that the disaggregations do not appear correctly
![image](https://github.com/open-sdg/open-sdg/assets/118024145/fef3e7bd-c32f-441a-999e-b4d5fd9105ef)

**Comment by brockfanning** [comment]
@SandraMarq Just to confirm, which version of Open SDG are you using right now? That was supposed to have been fixed in version 2.2.0.

**Reply by SandraMarq** [reply]
Hello

I made the corresponding updates for version 2.2.0

**Reply by SandraEliz** [reply]
Hello

I continue with the same details.

Still no solution?

Thank you

**Comment by brockfanning** [comment]
@SandraEliz Can you provide links to your data files?

**Reply by SandraMarq** [reply]
https://sdg-odsmexico.github.io/site/7-1-1/

**Comment by brockfanning** [comment]
@SandraMarq The problem is that your data_start_values includes a value for "Entidad federativa", but that is a geographical field and doesn't make sense for starting values on a map. I think it should be possible Open SDG to automatically ignore geographical fields in the starting values, so I'll open a ticket to try to fix this. However at the moment, it appears that starting values will not work on maps if they include geographical field values.

**Reply by brockfanning** [reply]
FYI I opened #2062
